<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SyncAccountFullMessage</class>
 <widget class="QWidget" name="SyncAccountFullMessage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>192</width>
    <height>105</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="wOQError">
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>10</number>
      </property>
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="topMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <property name="bottomMargin">
       <number>10</number>
      </property>
      <item>
       <widget class="QLabel" name="lOQWarning">
        <property name="text">
         <string>Your MEGA account is full. Uploads are disabled and folder synchronisation is paused.</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
        <property name="openExternalLinks">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignLeft">
       <widget class="QPushButton" name="bBuyMoreSpace">
        <property name="text">
         <string>Purchase more storage</string>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
